drop-config/src/lib.rs

```diff
@@ -11,6 +11,7 @@ pub struct DropConfig {
     pub dir_depth_limit: usize,
     pub transfer_file_limit: usize,
     pub storage_path: String,
+    pub checksum_events_size_threshold: Option<usize>,
```

Comment:
Can you document what `None` means in this case?

Reply:
Sure, I will add a comment. Responding to the question - apps asked for a configurable option for the threshold, so not to introduce a breaking change, there's an Option as it might not even be there.